tests/installation/releasenotes.pm

```diff
@@ -21,6 +21,12 @@ sub run() {
         record_soft_failure 'workaround missing release notes';
         return;
     }
+
+    # workaround for bsc#1014178
+    if (check_screen('zfcp-popup')) {
```

Comment:
this will waste 30 seconds in every run *unless* the screen is shown. Either make this a `check_screen …, 0;` or use a multi-tag assert_screen in a later step